peps/pep-0753.rst

```diff
@@ -208,16 +208,45 @@ as well as equivalent aliases.
 
 The following table lists these labels, in canonical form:
 
-.. csv-table::
-    :header: "Label", "Description", "Aliases"
-    :widths: 20, 50, 30
-
-    "``homepage``", "The project's home page", "*(none)*"
-    "``download``", "A download URL for the current distribution, equivalent to ``Download-URL``", "*(none)*"
-    "``changelog``", "The project's changelog", "``changes``, ``releasenotes``, ``whatsnew``, ``history``"
-    "``documentation``", "The project's online documentation", "``docs``"
-    "``issues``", "The project's bug tracker", "``bugs``, ``issue``, ``bug``, ``tracker``, ``report``"
-    "``sponsor``", "Sponsoring information", "``funding``, ``donate``, ``donation``"
+.. list-table::
+   :header-rows: 1
+
+   * - Label
+     - Description
+     - Aliases
+     - Human-readable equivalent
+   * - ``homepage``
+     - The project's home page
+     - *(none)*
+     - Homepage
+   * - ``source``
+     - The project's hosted source code or repository
+     - ``repository``, ``sourcecode``, ``github``
+     - Source Code
+   * - ``download``
+     - A download URL for the current distribution, equivalent to ``Download-URL``
+     - *(none)*
+     - Download
+   * - ``changelog``
+     - The project's changelog
+     - ``changes``, ``releasenotes``, ``whatsnew``, ``history``
+     - Changelog
+   * - ``documentation``
+     - The project's online documentation
+     - ``docs``
+     - Documentation
+   * - ``issues``
+     - The project's bug tracker
+     - ``bugs``, ``issue``, ``tracker``, ``issuetracker``, ``bugtracker``
+     - Issue Tracker
+   * - ``funding``
+     - Funding Information
+     - ``sponsor``, ``donate``, ``donation``
+     - Funding
```

Comment:
CC @hugovk -- this includes changes based on the data you highlighted in https://discuss.python.org/t/pep-753-uniform-project-urls-in-core-metadata/62792/6 🙂.